controllers/news.controller.js

```diff
@@ -0,0 +1,11 @@
+const { receiveAllTopics } = require("../modulles/news.modules");
+
+exports.getAllTopics = (req, res) => {
+  receiveAllTopics()
+    .then((data) => {
+      return data.rows;
```

Comment:
You have destructuring in your tests but not here. I'd try and stick to one style, either or is fine. But maybe not both. By putting ```{rows}``` in the ```.then(({rows})=>{})``` you could just return rows.